app/assets/stylesheets/application.css.scss

```diff
@@ -13,3 +13,34 @@
  *= require_tree .
  *= require_self
  */
+
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+@import url(https://fonts.googleapis.com/css?family=Quattrocento+Sans);
+
+
+html {
+  position: relative;
+  min-height: 100%;
+}
+
+.wrap {
+  min-height: 100%;
+  height: auto !important;
+  height: 100%;
+  margin: 0 auto -60px;
+}
+
+#push {
+  margin-bottom: 362px;
+}
+
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  /* Set the fixed height of the footer here */
+  height: 60px;
+  background-color: #f5f5f5;
+}
```

Comment:
New line plz. You can also have Sublime Text do this for automatically by adding

```json
"ensure_newline_at_eof_on_save": true
```
to your User preferences file.